src/components/pages/ProfilePage/index.jsx

```diff
@@ -84,9 +129,32 @@ class ProfilePage extends React.Component {
 }
 //#endregion
 
+//#region > Redux Mapping
+const mapStateToProps = (state) => ({
+  loggedUser: state.auth.loggedUser,
+  fetchedUser: state.user.fetchedUser,
+  cachingDone: state.user.cachingDone,
+});
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    saveSettings: (nextSettings) => dispatch(saveSettingsActions(nextSettings)),
+    readCache: (username) => dispatch(readCacheAction(username)),
+    updateCache: (fetchedUser) => dispatch(updateCacheAction(fetchedUser)),
+  };
+};
+//#endregion
+
 //#region > Exports
-//> Default Class
-export default withRouter(ProfilePage);
+/**
+ * Provides its connected component with the pieces of the data it needs from
+ * the store, and the functions it can use to dispatch actions to the store.
+ * Got access to the history object’s properties and the closest
+ * <Route>'s match.
+ */
```

Comment:
```suggestion
/**
 * Provides its connected component with the pieces of the data it needs from
 * the store, and the functions it can use to dispatch actions to the store.
 *
 * Got access to the history object’s properties and the closest
 * <Route>'s match.
 */
```